content/help/faq/security/check-policy.md

```diff
@@ -6,9 +6,11 @@ weight: 11
 The `istioctl` tool provides an option for this purpose. You can do:
 
 {{< text bash >}}
-$ istioctl authn tls-check httpbin.default.svc.cluster.local
+$ istioctl authn tls-check $CLIENT_POD httpbin.default.svc.cluster.local
 HOST:PORT                                  STATUS     SERVER     CLIENT     AUTHN POLICY        DESTINATION RULE
 httpbin.default.svc.cluster.local:8000     OK         mTLS       mTLS       default/            default/default
```

Comment:
Isn't the default destination rule `istio-system/default`?

Reply:
I change this to istio-system to make it less confusing. It's still an example, so we don't know what setup users actually have so either way should be fine, I think.

Reply:
Unless the task creates a destination rule in the default namespace, which it doesn't, the Istio default one is used and `istio-system` is the default Istio namespace. So, using anything else here would be confusing.

Reply:
Done